.github/workflows/csound_builds.yml

```diff
@@ -251,6 +251,56 @@ jobs:
 
       - name: Run integration and unit tests
         run: cmake --build build --target test csdtests
+  
+  coverage_build:
+    name: Coverage build (linux/vcpkg)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - name: Export GitHub Actions cache environment variables
+        uses: actions/github-script@v6
+        with:
+          script: |
+            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
+
+      - name: Bootstrap VCPKG
+        run: ./vcpkg/bootstrap-vcpkg.sh
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install cmake libjack-dev libpulse-dev ladspa-sdk dssi-dev autoconf libtool
+
+      - name: Configure Build
+        run: cmake -B build -S . -DUSE_VCPKG=1 -DCUSTOM_CMAKE="./platform/linux/custom.cmake" -DTRACK_COVERAGE=1
+
+      - name: Build Csound
+        run: cmake --build build --config Release
+
+      - name: Run integration and unit tests
+        run: cmake --build build --target test csdtests
```

Comment:
We don't need to run tests on this. It will have already run for the normal Linux build. Remove any step that is redundant

Reply:
We need to run tests after building with `-DTRACK_COVERAGE=1`. I could add `-DTRACK_COVERAGE=1` to the normal Linux build though?

Reply:
Why?

Reply:
The point of coverage is to tell you which lines of code were run during testing?

Reply:
No way really? 😄 

Gcov is a static analysis though? You feed in the source files and it gives you the result. That's irrespective of running the tests or not.

Reply:
https://github.com/shenxianpeng/gcov-example Ok i see how it works now. So there are two parts to it. That makes more sense to me now. Looks good so